lib/binary-build.js

```diff
@@ -5,9 +5,7 @@ const logger = require('./common-log')();
 
 function wait (timeout) {
   return new Promise((resolve) => {
-    setTimeout(() => {
```

Comment:
you wrote this code :) :)

Reply:
I am often almost immediately embarrassed by the code I write.